projects/plugins/jetpack/extensions/blocks/subscriber-login/block.json

```diff
@@ -4,20 +4,32 @@
 	"name": "jetpack/subscriber-login",
 	"title": "Subscriber Login",
 	"description": "Subscriber Login",
-	"keywords": [],
+	"keywords": [ "login", "logout", "subscription", "member", "account", "user" ],
 	"version": "0.0.1",
 	"textdomain": "jetpack",
 	"category": "grow",
 	"icon": "<svg viewBox='0 0 32 32' width='32' height='32' xmlns='http://www.w3.org/2000/svg'><path d='M9 15h2V9H9v6zm1-10c-.5 0-1 .5-1 1s.5 1 1 1 1-.5 1-1-.5-1-1-1zm0-4c-5 0-9 4-9 9s4 9 9 9 9-4 9-9-4-9-9-9zm0 16c-3.9 0-7-3.1-7-7s3.1-7 7-7 7 3.1 7 7-3.1 7-7 7z'/></svg>",
 	"supports": {
-		"align": false,
-		"alignWide": true,
+		"align": true,
 		"anchor": false,
 		"customClassName": true,
 		"className": true,
 		"html": false,
 		"multiple": true,
-		"reusable": true
+		"reusable": true,
+		"spacing": {
+			"margin": true,
+			"padding": true
+		},
+		"typography": {
```

Comment:
Ah, the experimental ones are _so much_ nicer as they don't clutter the UI, but I suppose we can use these meanwhile.

https://github.com/WordPress/gutenberg/blob/60a9413a1f438b424254ec0d88f74745b9a60bf4/packages/block-library/src/loginout/block.json#L33-L41

<img width="281" alt="Screenshot 2024-01-19 at 12 26 28" src="https://github.com/Automattic/jetpack/assets/87168/14748416-1b49-476d-b1c7-791c72947b23">

Reply:
I was thinking of using it but for me "experimental" means it can be buggy 😄

Reply:
Yep all good; they're not buggy per-se but not a stable API and could still change between WP releases.